owlbot.py

```diff
@@ -17,19 +17,31 @@
 # This code snippet can be used to manually update the typeless bot
 # to a different version than OwlBot has baked in, but this shouldn't
 # be needed once it's settled down.
-"""
 import os
 from synthtool import shell
 from synthtool.log import logger
 old_path = os.getcwd()
 os.chdir("/synthtool")
-logger.debug("Update typeless sample bot [1.1.0]")
-shell.run(["npm", "i", "@google-cloud/typeless-sample-bot@1.1.0"])
+logger.debug("Update typeless sample bot [1.3.0]")
+shell.run(["npm", "i", "@google-cloud/typeless-sample-bot@1.3.0"])
```

Comment:
I thought we built `@google-cloud/typeless-sample-bot` into the post-processor, why is this install necessary?

Reply:
Mentioned in Karl's issue on the main bot repo, but I was basically saving review/release time looping through synthtool. It's my intent to get synthtool updated once it's fully stabilized, and I can remove this in another PR.